_layouts/getstarted.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: base
+---
+
+<div class="grid-wrapper getstarted-page">
+    <div class="grid__item width-12-12 width-12-12-mobile">
+        <h1 class="text-caps">{{page.title}} {{page.docversion}}</h1>
+    </div>
+    <div class="grid__item width-12-12 width-12-12-mobile">
+        <div>
+            {{ content }}
+        </div>
+    </div>
+    <div class="grid__item width-12-12 width-12-12-mobile">
+        <div>&lt; <a href="{{ site.baseurl }}/">Back</a></div>
+    </div>
```

Comment:
Other pages don't have this and I don't think this one should. Just because you can go to this page from any other page, as it's is linked from the standard header/footer. So if you do that, this is not a 'Back' link, it's a 'Home' link.

Reply:
Yes, I agree. I'm going to change that to "Home"